src/cluster/src/helm.rs

```diff
@@ -127,6 +127,37 @@ impl HelmClient {
             .count();
         Ok(count > 0)
     }
+
+    /// Returns the list of installed sys charts
+    pub(crate) fn installed_sys_charts(&self, name: &str) -> Result<Vec<InstalledChart>, IoError> {
```

Comment:
this API doesn't belong in the helm client since helm client should be generic and not have any fluvio logic